_includes/payment-resource.md

```diff
@@ -90,7 +91,21 @@ Content-Type: application/json
             "href": "{{ page.apiUrl }}/psp/{{ payment-instrument }}/payments/{{ page.paymentId }}/captures",
             "rel": "create-capture",
             "contentType": "application/json"
+        },
```

Comment:
This trailing comma should not be here if the `showStatusOperations` is `false`. I think you need to move `
{% if showStatusOperations == true %}` up to line 94 between `}` and `,`.

```suggestion
        }{% if showStatusOperations == true %},
```